DigitalLearningSolutions.Web/Models/AdminLoginDetails.cs

```diff
@@ -43,7 +43,7 @@ public AdminLoginDetails(AdminUser adminUser)
             PublishToAll = adminUser.PublishToAll;
             SummaryReports = adminUser.SummaryReports;
             IsUserAdmin = adminUser.IsUserAdmin;
-            CategoryId = adminUser.CategoryId;
+            CategoryId = adminUser.CategoryId ?? 0;
```

Comment:
It'd be nice to extract this into a helper method for mapping `AdminCategory => CategoryId` and do the same for `CategoryId => AdminCategory`, then use those methods everywhere we convert between the two. Then it's possible to find everywhere we're doing the conversion by looking for usages of those methods.